bftengine/src/preprocessor/tests/preprocessor_test.cpp

```diff
@@ -127,7 +127,7 @@ class DummyReplica : public InternalReplicaApi {
   SeqNum getPrimaryLastUsedSeqNum() const override { return 0; }
   uint64_t getRequestsInQueue() const override { return 0; }
   SeqNum getLastExecutedSeqNum() const override { return 0; }
-
+  void registerStopCallBack(std::function<void(void)> f) override { (void)f; };
```

Comment:
1)  CallBack is one word and can be written Callback, so registerStopCallBack->registerStopCallback
2) The right way in to overcome unused in C++17 is with [[maybe_unused]]:
void registerStopCallBack([[maybe_unused]] std::function<void(void)> f) override {};

Reply:
1. will do
2. it's in a test. in production I'd like it to be used for sure

Reply:
done